editor/editor_node.h

```diff
@@ -481,6 +481,7 @@ class EditorNode : public Node {
 
 	String import_reload_fn;
 
+	HashSet<String> force_textfile_extensions;
```

Comment:
Not sure if this kind of future-proofing is worth it now. It could be hard-coded to CSV, with a comment to add a HashSet if more extensions are needed.

Though either way is fine probably.